mobify/platform-scaffold
# Rename containers to .JSX file extension

**JIRA**: small part of https://mobify.atlassian.net/browse/WEB-1224
 **Linked PRs**: none

## Changes
- Rename all files that contain JSX to have `.jsx` file extension
- Necessary config changes to make webpack happy as a result of this change
- Fix an unrelated bug in PLP where we don't detect the "no products for category" situation correctly and instead just load indefinitely

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- Verify everything works as per usual
- Go to a PLP with no products in the category such as "new arrivals". Previously this would have showed skeletons forever, now it should show the no results message.
